packages/manager/modules/dbaas-logs/src/detail/account/setup/logs-account-setup.scss

```diff
@@ -0,0 +1,7 @@
+dbaas-logs-detail-account-setup {
+  @import 'bootstrap4/scss/_functions.scss';
```

Comment:
looks like these bootstrap files are imported many times, 18 times. can you please import them once maybe in root css file?

Reply:
Each component should be independent and should be re-usable without any dependencies on the parent. That is the reason why the import is in each module. What do you think?

Reply:
This is done :)